src/components/layout/WithVehicleReference.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import LayoutWithVehicleReference from './WithVehicleReference.tsx';
+import Stack from '../stack';
+import Input from '../input';
+
+<Meta
+  title="Layout/Pages/Layout with vehicle reference"
+  component={LayoutWithVehicleReference}
+/>
+
+export const Template = (args) => <LayoutWithVehicleReference {...args} />;
+
+# Layout with vehicle reference
+
+## With back link
+
+<Canvas>
+  <Story
+    name="With back link"
+    args={{
+      title: 'Nachricht an den Verkäufer',
+      backLink: {
+        text: 'Back',
+        url: 'https://www.autoscout24.ch/de',
+      },
+      vehicle: {
+        vehicleTitle: 'AUDI A5 Sportback 3.0 TDI quattro S-tronic (Limousine)',
+        price: `CHF 23'900.–`,
+        sellerName: `Auto-Center Grenchen AG 2540 Grenchen (SO)`,
+        sellerAddress: `2540 Grenchen (SO)`,
+        image: <img src="/dummyImage.jpeg" alt="vehicle reference image" />,
+      },
+      children: (
+        <Stack spacing="md">
+          <Input name="name" placeholder="Name" />
+          <Input name="email" placeholder="E-Mail address" />
+          <Input name="phone" placeholder="Phone number" />
+        </Stack>
+      ),
```

Comment:
You can add `args` and `argTypes` on `Meta`, this way you won't need to repeat them in every story, but just overwrite the ones that you need.